.circleci/config.yml

```diff
@@ -78,19 +78,24 @@ jobs:
       - run:
           name: Set up Rust
           command: |
+            apt update
+            apt install build-essential curl libstdc++6 libstdc++-10-dev libssl-dev pkg-config -y
+            apt install cmake -y
             curl https://sh.rustup.rs -sSf | sh -s -- -y
             export PATH=$PATH:$HOME/.cargo/bin
             echo 'export PATH=$PATH:$HOME/.cargo/bin' >> $BASH_ENV
             rustc --version
-            cargo build
+            cargo build --features=bigtable
```

Comment:
Getting late in the week but something almost concerns me about this differing from what we build elsewhere -- I think solely in the Dockerfile -- so we're not testing exactly what we produce as an artifact. I imagine we'll enable there shortly anyway so maybe not a big deal for now.

Reply:
Yeah, I see this as a temporary thing. `dual_data` already segments all of the storage methods under feature flags (so that you have `bigtable`, `dynamodb`, `dual` and the `default` points to `dynamodb`.) Once things stabilize, we can alter the `default`